app/models/ems_event.rb

```diff
@@ -132,6 +132,20 @@ def self.first_chained_event(ems_id, chain_id)
     EmsEvent.where(:ems_id => ems_id, :chain_id => chain_id).order(:id).first
   end
 
+  def self.filter_args_for_add(args)
+    return args unless args.last.kind_of?(Hash)
+
+    if %w[EMBEDDEDANSIBLE ANSIBLETOWER].include?(args.last[:source])
```

Comment:
If we don't want to log extra_vars could we do that in the ansible_tower event_parser before it even hits `EmsEvent.add_queue` ?

Reply:
E.g. here, https://github.com/ManageIQ/manageiq-providers-ansible_tower/blob/master/app/models/manageiq/providers/ansible_tower/automation_manager/event_parser.rb#L11

Reply:
@agrare if we do that, then we are stripping them from the hash entirely, instead of just filtering them.

Unfortunately, the logging happens both on a `MiqQueue.put` and `MiqQueue#deliver`, so I am unsure there is a better way to handle this (or at least in a way that my feeble little mind can come up with).

I am also somewhat sure that we take this `extra_vars` data and digest it as part of the refresh:

https://github.com/ManageIQ/manageiq-providers-ansible_tower/blob/master/app/models/manageiq/providers/ansible_tower/inventory/parser/automation_manager.rb#L38

or some other "ems digestion action", but if I am totally wrong on that, then I agree we can get rid of this and do this on the `manageig-providers-ansible_tower` end of things.

Reply:
Ah so [that](https://github.com/ManageIQ/manageiq-providers-ansible_tower/blob/master/app/models/manageiq/providers/ansible_tower/inventory/parser/automation_manager.rb#L38) isn't related to events, it hits the tower API to get the extra_vars and will save them in the `JobTemplate` model